peerdb-cloud/hipaa-eligible-services.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: PeerDB HIPAA Eligible Services
+description: "HIPAA Eligible Services provided by PeerDB"
+icon: "caduceus"
+iconType: "solid"
+
+---
+
+The following list of services are HIPAA Eligible and can be used for storing and processing Protected Health Information (PHI) after signing a Business Associate Agreement (BAA) with PeerDB:
+* [PeerDB Cloud](https://app.peerdb.cloud/) with the following restrictions:
+    * Mirror Types:
+      * CDC
+        * Source Peers (with minimum supported PeerDB version in parentheses):
+          * PostgreSQL (v0.15.1)
+        * Destination Peers (with minimum supported PeerDB version in parentheses):
+          * Snowflake (v0.15.1)
+          * BigQuery (v0.15.1)
+          * Postgres (v0.15.1)
+      * QRep
```

Comment:
should include queues

Reply:
For queues, the issue if the data is corrupted, can we do a resync? Or should we call [dropping mirror + target] and rekick-off as resync? In that case even CH would pass data restore test.

Reply:
if the task is just to get data back on the target, that should count
then everything would work as you said